answer/src/main/java/org/exoplatform/platform/qa/ui/answer/pageobject/AnswerManagement.java

```diff
@@ -27,12 +32,7 @@ public AnswerManagement(TestBase testBase) {
    */
   public void goToAnswerQuestion(String question) {
     info("Go to answer a question");
-
-    if (evt.waitForAndGetElement(ELEMENT_QUESTION_LIST_ITEM.replace("$question", question), 5000, 0) != null)
-      evt.click(ELEMENT_QUESTION_LIST_ITEM.replace("$question", question));
-    evt.waitForAndGetElement(ELEMENT_QUESTION_SELECTED_ITEM.replace("$question", question));
-    evt.click(ELEMENT_ANSWER_BUTTON);
-    evt.waitForAndGetElement(ELEMENT_ANSWER_FORM);
+    $(byText(question)).parent().parent().parent().find(byText("Answer")).click();
```

Comment:
The only attribute static in this element is the label. the xpath of the element which have label "Answer" is variable. I used byText to simplify the cases. Now I change it to use xpath (recover the id of question and use it in the xpath of answer)